### ISSUE TITLE
top-scope variable being used without an explicit namespace in a string with a lookup

### ISSUE THREAD

**Issue by biemond** [rodjek/puppet-lint#635]
Hi,

Using puppet-lint 2.1.1

I got this WARNING: top-scope variable being used without an explicit namespace on
when I have something like this 

````
  $exec_path         = "${jdk_home_dir}/bin:${lookup('orawls::exec_path')}"
````

Lookup is to my internal data module.

Thanks Edwin
